components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
-                        width="32"
-                    ></v-skeleton-loader>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
+                    <p class="d-block ma-1">Legenda:</p>
+                    <div class="ma-1 flex-wrap" style="width: 100%">
+                        <div v-if="showFeatures">
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#9400D3"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Polígonos Prioritários: Muito Alta</p>
+                                </v-col>
+                            </v-row>
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#FF0000"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Polígonos Prioritários: Alta</p>
+                                </v-col>
+                            </v-row>
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#FF8C00"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Polígonos Prioritários: Média</p>
+                                </v-col>
+                            </v-row>
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#FFD700"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Polígonos Prioritários: Baixa</p>
+                                </v-col>
+                            </v-row>
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#008000"
+                                    >mdi-square</v-icon
+                                >
+                                <v-col no-gutters cols="6">
+                                    <p>Polígonos Prioritários: Muito Baixa</p>
+                                </v-col>
+                            </v-row>
+                        </div>
+                        <div v-if="showFeaturesUrgentAlert">
+                            <v-row no-gutters align="center">
+                                <v-icon x-small color="#965213"
```

Comment:
My suggestion is to iterate over data to create these legends.

Reply:
I agree, my suggestion is to keep it this way to avoid refactoring because it will take time, this information will be returned from the back in the future, requiring intervention in the code anyway, at that moment the iteration will be done correctly, what do you think? if you prefer I can do it.